src/table/src/requests.rs

```diff
@@ -212,6 +214,33 @@ pub enum AlterKind {
     RenameTable {
         new_table_name: String,
     },
+    ChangeTableOptions {
+        options: Vec<ChangeTableOptionRequest>,
+    },
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum ChangeTableOptionRequest {
```

Comment:
We have a similar struct `ChangeRegionOption`. They have the same fields. Can we reuse the `ChangeRegionOption` struct?

Reply:
fixed in fixed in https://github.com/GreptimeTeam/greptimedb/pull/4848/commits/56f4be4c66137553ad3261fd1163e44d9939bdf7